### PR TITLE
Makman2/stuff

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2151/commits/19da95999a9e5393f291e06778ea282cb7ad540e https://github.com/coala-analyzer/coala/pull/2151/commits/b6869bede5334d5dcdc100d04280300e2f49db9c

**Comment by sils1297** [comment]
needs a rebase, cancelled build for that

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2151/commits/4a94c55db70403a8e512f131453410811d1dc5a6

**Comment by Makman2** [comment]
reack b405aa8, 6049540

File: coalib/bearlib/abstractions/Linter.py [hunk 1]
```diff
@@ -277,8 +277,8 @@ def _convert_output_regex_match_to_result(self,
 
             # Construct the result.
             return Result.from_values(
-                origin=groups.get("origin", self),
-                message=groups.get("message", ""),
+                origin=groups.get("origin", self).strip(),
```
Comment (sils1297) [inline coalib/bearlib/abstractions/Linter.py]: you can't strip self :)

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala/pull/2151/commits/4a94c55db70403a8e512f131453410811d1dc5a6

File: coalib/misc/Shell.py [hunk 2]
```diff
@@ -137,24 +137,3 @@ def prepare_string_argument(string, shell=get_shell_type()):
         return string
```
Comment (sils1297) [inline coalib/misc/Shell.py]: newlines should be removed as well, huh?

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2151/commits/60495403c9261f3ea8158213d5bf1330a6eb9b48 https://github.com/coala-analyzer/coala/pull/2151/commits/6ac31ec7405439438e33b436ed05c3a3f8d64ae6 https://github.com/coala-analyzer/coala/pull/2151/commits/6a8777f04750bac7f8363a098be208f25e6664ff https://github.com/coala-analyzer/coala/pull/2151/commits/400d4692d8db0edfacde5380a2e5f88921b76297 https://github.com/coala-analyzer/coala/pull/2151/commits/b1b71e92fe217ae5dd22f44626b7cf4a82bb1db3

File: coalib/misc/Shell.py (hunk 2 above)
Comment (sils1297) [inline coalib/misc/Shell.py]: heh, gitmate doesn't like it either

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2151/commits/95816ac1ffa34df3ed65e87768f0e6b1fdd92813  https://github.com/coala-analyzer/coala/pull/2151/commits/dfd870ba0b489e65784cbbb95295dd0ba09373c5

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**